sdk/ml/azure-ai-ml/azure/ai/ml/entities/_job/automl/tabular/forecasting_job.py

```diff
@@ -354,6 +354,68 @@ def set_training(
         blocked_training_algorithms: Optional[List[str]] = None,
         training_mode: Optional[Union[str, TabularTrainingMode]] = None,
     ) -> None:
+        """
+        The method to configure forecast training related settings.
+
+        :keyword enable_onnx_compatible_models:
```

Comment:
Please use the tag :param: for example:
` :param enable_onnx_compatible_models:`